THANKS.txt

```diff
@@ -243,6 +243,7 @@ Shashaank N for contributions to scipy.signal.
 Frank Torres for fixing a bug with solve_bvp for large problems.
 Ben West for updating the Gamma distribution documentation.
 Terry Davis for documentation improvements in scipy.ndimage.morphology
+Denali Molitor for documentation edits in lsmr.
```

Comment:
I think we're switching to a workflow where `THANKS` entries are added here after a PR is merged: 
https://github.com/scipy/scipy/wiki/*THANKS.txt-additions-modifications-for-1.6.0*

Mostly to avoid merge conflicts I think

Reply:
Thanks. I have reverted the edit to THANKS.txt